### DISCUSSION TITLE
Navicat Premium Fix

### DISCUSSION THREAD

**Discussion by XEmpty** [elrumo/macOS_Big_Sur_icons_replacements#1235] [Icon request]
Hi, can someone fix "Navicat Premium" & "Krita" icons . Border so pixelated and it's looks ugly . ( pixelated border in "Krita" you will only see in launchpad )
Ty for help!

**Comment by QuintB** [comment]
Hi there, you can download both of these from [my profile](https://macosicons.com/u/Quint) until they're approved.
![Navicat](https://user-images.githubusercontent.com/50681031/126881902-c3453b9f-6395-4926-96c6-ea660d5f95dc.png)
![Krita](https://user-images.githubusercontent.com/50681031/126881904-a570bce3-baa1-4bff-9358-935915b6fb79.png)
